automatic/virtualbox_5.0.x/virtualbox_5.0.x.nuspec

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata>
+    <id>virtualbox</id>
```

Comment:
Is using `_` in package ID considered best practies (eg. over `-`)?
And do we need the `.x` in the package name?